lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -59,9 +66,29 @@ export async function updateArtifacts({
         constraint: config.constraints?.kustomize,
       });
     }
-    await exec(`helmfile deps -f ${quote(packageFileName)}`, {
+
+    const cmd: string[] = [];
+    const doc = parseDoc(newPackageFileContent);
+
+    const regexOfURLPath = /\/.*/;
+    const repositories = getRepositories(doc);
+
+    for (const value of repositories.filter(isOCIRegistry)) {
+      const loginCmd = generateRegistryLoginCmd(
+        value.name,
+        'https://' + value.url,
+        value.url.replace(regexOfURLPath, '')
```

Comment:
```suggestion
        replaceUrlPath(value.url, "")
```
Use the http utils

Reply:
this is not work with `Invalid URL`, bacause of the value.url contains like ghcr.io/private-charts.